### PR TITLE
fix(Dockerfile): use foreman instead of rails server

### PR DESCRIPTION
**Pull request by orlando**
**What:** use foreman instead of rails server

**Why:** Make sure we run the application on production same as we do in development

**How:**

- Add foreman to Dockerifle

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,14 +18,13 @@ WORKDIR $APP_HOME
...
+  gem install bundler foreman
 
 RUN bundle install
 RUN yarn install --check-files
 
 ADD . $APP_HOME
 
-EXPOSE 3000
-ENV RAILS_ENV production
+EXPOSE 5000
```
Comment (castrolem) [inline Dockerfile:29]: Why the port change? Locally it seems to run on 3000 too when running the Procfile file with foreman.

Comment (orlando) [reply Dockerfile:29]: default is 5000 for foreman. https://github.com/ddollar/foreman/issues/667#issue-206601005

Comment (castrolem) [reply Dockerfile:29]: Interesting, then maybe we need to change `puma.rb`, because we have `port        ENV.fetch("PORT") { 3000 }`

Comment (orlando) [reply Dockerfile:29]: foreman users $PORT environment variable. I'll set that in the Dockerfile

**Review by castrolem — APPROVED** [review]
Looks good to me.

**Pull request merged**